docs/Animations.md

```diff
@@ -0,0 +1,238 @@
+---
+id: animations
+title: Animations
+layout: docs
+category: Guides
+permalink: docs/animations.html
+next: nativemodulesios
+---
+
+Fluid, meaningful animations are essential to the mobile user
+experience. These are the APIs that are currently made available by
+React Native to assist you in creating these animations. A new API that
+will replace most of these and/or use them under the hood is under development
+and can be expected sometime in July or August.
+
+### JavaScript-based
+
+#### react-tween-state
+
+[react-tween-state](https://github.com/chenglou/react-tween-state) is a
+minimal library that does exactly what its name suggests: it *tweens* a
+value in a component's state, starting at a **from** value and ending at
+a **to** value. This means that it generates the values in between those
+two values, and it sets the state on every `requestAnimationFrame` with
+the intermediary value.
+
+```
```

Comment:
I did this at first but it is quite ugly when it's red -- it seems like it's a warning. Maybe I need to look at adding a new style to the docs that handle quotes in a better way.